go.mod

```diff
@@ -205,3 +205,7 @@ require (
 
 // We're replacing kryptology with our own fork, which fixes dependencies' security vulnerabilities.
 replace github.com/coinbase/kryptology => github.com/ObolNetwork/kryptology v0.0.0-20231016091344-eed023b6cac8
+
+// We're replacing go-eth2-client with a branch off our fork, at version v0.21.10.
+// This is needed to ensure Gnosis compatibility.
+replace github.com/attestantio/go-eth2-client => github.com/ObolNetwork/go-eth2-client v0.21.11-0.20240822135044-f0a5b21e02c6
```

Comment:
Wouldn't that mean we will have to manage this repo constantly and rebase it every time there is a new version of go-eth2-client?

Reply:
Hopefully go-eth2-client will fix this upstream soon